docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+   domain1-managed-server2            ClusterIP      None          <none>           8001/TCP             2d20h
+   internal-weblogic-operator-svc     ClusterIP      10.0.192.13   <none>           8082/TCP             2d22h
+   kubernetes                         ClusterIP      10.0.0.1      <none>           443/TCP              2d22h
+   ```
+
+   In the example, the URL to access the Administration Server is: `http://52.188.176.103:7001/console`.  The default user name for the Administration Console is `weblogic` and the default password is `welcome1`.  Please change this for production deployments.
+
+   If the WLS Administration Console is still not available, use `kubectl describe domain` to check domain status.
+
+   ```bash
+   $ kubectl describe domain domain1
+   ```
+
+   Make sure the status of cluster-1 is `ServersReady` and `Available`.
+   {{%expand "Click here to view the example status." %}}
+   ```yaml
+   Status:
+    Clusters:
+      Cluster Name:      cluster-1
+      Maximum Replicas:  5
+      Minimum Replicas:  1
+      Ready Replicas:    2
+      Replicas:          2
+      Replicas Goal:     2
+    Conditions:
+      Last Transition Time:  2020-07-06T05:39:32.539Z
+      Reason:                ServersReady
+      Status:                True
+      Type:                  Available
+    Replicas:                2
+    Servers:
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    admin-server
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    managed-server1
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    managed-server2
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server3
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server4
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server5
+   ```
+   {{% /expand %}}
+
+To deploy a sample application on WLS, you may skip to the section [Deploy sample application](#deploy-sample-application).  The next section includes a script that automates all of the preceding steps.
+
+#### Automation
+
+If you want to automate the above steps of creating AKS cluster and WLS domain, you can use the script `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks.sh`.
+
+The sample script will create a WLS domain home on the AKS cluster, including
+  - creating a new Azure resource group, with a new Azure Storage Account and Azure File Share to allow WebLogic to persist its configuration and data separately from the Kubernetes pods that run WLS workloads.
```

Comment:
creating  -> Creating (please initial capitalize the first word of each of the following bulleted items)

Reply:
Will do.